VotingApplication/VotingApplication.Web.Api/Scripts/RankedVote.js

```diff
@@ -242,121 +244,6 @@
-            $("#chart-results").html('');
-            $("#chart-buttons").html('');
-
-            if (!self.chartVisible() || data.length === 0)
-                return;
-
-            //Get voter count
-            var voterCount = 0;
-            data[0].forEach(function (d) {
-                voterCount += d.Voters.length;
-            });
-
-            chart = new insight.Chart('', '#chart-results')
-            .width($("#chart-results").width())
-            .height($("#chart-results").width());
-
-            var xAxis = new insight.Axis('', insight.scales.ordinal)
-                .isOrdered(true)
-                .orderingFunction(function (a, b) {
-                    var finalAIndex = orderedNames.indexOf(a.Name);
-                    var finalBIndex = orderedNames.indexOf(b.Name);
-                    return finalAIndex - finalBIndex;
-                })
-                .tickLabelRotation(45);
-
-            var yAxis = new insight.Axis('Votes', insight.scales.linear)
-                .tickFrequency(1)
-                .axisRange(-0.1, voterCount);
-            chart.xAxis(xAxis);
-            chart.yAxis(yAxis);
-            chart.legend(new insight.Legend());
-
-            chart.series([]);
-
-            var seriesIndex = roundIndex;
-
-            //Add a button to display individual rounds
-            for (var i = 1; i <= data.length; i++) {
-                $("#chart-buttons").append('<button class="btn btn-primary" onclick="self.filterRounds(' + i + ')">Round ' + i + '</button>');
-            }
-
-            //Disable the currently selected button
-            var currentRoundButton = $("#chart-buttons").children()[roundIndex];
-            $(currentRoundButton).attr('disabled', 'disabled');
-
-            //Filter to a specific round
-            if (roundIndex > 0) {
-                data = data.slice(roundIndex - 1, roundIndex);
-                seriesIndex = roundIndex - 1;
-            }
-
-            //Map out each round
-            data.forEach(function (roundData) {
-
-                var voteData = new insight.DataSet(roundData);
-                var series = new insight.ColumnSeries('votes_' + (seriesIndex++), voteData, xAxis, yAxis)
-                .keyFunction(function (d) {
-                    return d.Name;
-                })
-                .valueFunction(function (d) {
-
-                    return d.BallotCount;
-                })
-                .title('Round ' + seriesIndex)
-                .tooltipFunction(function (d) {
-                    if (d.Voters.length > 0) {
-                        return d.Voters.toString().replace(/,/g, "<br />");
-                    }
-                    else
-                        return "Option eliminated";
-                });
-
-                var newSeries = chart.series();
-                newSeries.push(series);
-                chart.series(newSeries);
-            });
-
-            //Annotate the decision line
```

Comment:
The annotation is important for the Ranked poll. We should still be able to add a line at voterCount/2.